### ISSUE TITLE
Please include nqp-configure in the release tarballs

### ISSUE THREAD

**Issue by pstumpf** [Raku/nqp#699]
As the title says.  As a downstream packager, this would make my work a lot easier.

**Comment by patrickbkr** [comment]
@pstumpf Is this really about the submodule in the NQP source release files? (Which I suspect few people actually know about.) Or is this actually about the rakudo source release files? (The ones linked to on the <http://rakudo.org/downloads> website.) 

@Altai-man If I understand this ticket correctly it's basically a matter of adapting the release procedure. Isn't this ticket then better solved by you?

**Comment by pstumpf** [comment]
@patrickbkr Yes, this is about the source releases generated from this (Raku/nqp) github repository.

**Comment by vrurg** [comment]
This issue is confusing me now. I just have checked out rakudo sources available via download on rakudo.org and found `nqp-configure` included. Then downloaded https://github.com/Raku/nqp/releases/download/2021.04/nqp-2021.04.tar.gz from this repo and it contains the submodule too. 2021.03 is no different.

It currently looks like it's harder to find a source tarball without nqp-configure than with it included. @pstumpf could you, please, elaborate as to where do you download from?

**Comment by Altai-man** [comment]
@vrurg I think the ticket is about "nqp-2021.04.tar.gz" we generate in this repo.
It boils down to `make release` in nqp repo not skipping the `nqp-configure` directory, I'll look into it.

**Comment by patrickbkr** [comment]
@pstumpf Did you possibly download the archive labeled "[Source Code (ext)](https://github.com/Raku/nqp/archive/refs/tags/2021.04.tar.gz)"? Those are auto-generated by GitHub for every package and are best ignored. That's a plain bundle of the files of the respective tag.

**Comment by patrickbkr** [comment]
@Altai-man IIU vrurg correctly, those archives already contain nqp-configure. So the archives should already be fine. Question is: Can we do something about people downloading the auto-generated "Source Code" archives? Is it possible to add some descriptive text to the GitHub package? If yes we could add a short warning: "The `nqp-*.tar.gz` file contains the packaged source code of the release. Please ignore the `Source Code (ext)` files, those are automatically generated by GitHub ant contain the unprepared files of the repository."

**Comment by Altai-man** [comment]
Hmm, actually you are right, `nqp-configure` is present in the release tarball for this repo.
@pstumpf can you please clarify?
